packages/flutter_test/test/matchers_test.dart

```diff
@@ -363,6 +363,15 @@ void main() {
         expect(comparator.golden, Uri.parse('foo.png'));
       });
 
+      testWidgets('if comparator succeeds using Roboto', (WidgetTester tester) async {
+        await tester.pumpWidget(boilerplate(const Text('hello')));
+        final Finder finder = find.byType(Text);
+        await expectLater(finder, matchesGoldenFile('foo.png', useRoboto: true,));
```

Comment:
I don't think we have golden file tests in the flutter_test package currently. I am surprised this isn't failing.